README.md

```diff
@@ -4,68 +4,68 @@ jrnl [![Build Status](https://travis-ci.com/jrnl-org/jrnl.svg?branch=master)](ht
 _To get help, [submit an issue](https://github.com/jrnl-org/jrnl/issues/new) on
 Github._
 
-*jrnl* is a simple journal application for your command line. Journals are
-stored as human readable plain text files - you can put them into a Dropbox
-folder for instant syncing and you can be assured that your journal will still
-be readable in 2050, when all your fancy iPad journal applications will long be
-forgotten.
+`jrnl` is a simple journal application for the command line.
 
-Optionally, your journal can be encrypted using the [256-bit
-AES](http://en.wikipedia.org/wiki/Advanced_Encryption_Standard).
+Its goal is to facilitate the rapid creation and viewing of journal entries. It
+is flexible enough to support different use cases and organization strategies.
+It is powerful enough to search through thousands of entries and display, or
+"filter," only the entries you want to see.
 
-### Why keep a journal?
+`jrnl` includes support for [128-bit AES
+encryption](http://en.wikipedia.org/wiki/Advanced_Encryption_Standard) using
+[cryptography.Fernet](https://cryptography.io/en/latest/fernet/).
 
-Journals aren't just for people who have too much time on their summer
-vacation. A journal helps you to keep track of the things you get done and how
-you did them. Your imagination may be limitless, but your memory isn't. For
-personal use, make it a good habit to write at least 20 words a day. Just to
-reflect what made this day special, or why you haven't wasted it. For
-professional use, consider a text-based journal to be the perfect complement to
-your GTD todo list - a documentation of what and how you've done it.
+## In a Nutshell
 
-In a Nutshell
--------------
+To make a new entry, just enter
 
-To make a new entry, just type
+``` sh
+jrnl yesterday: Called in sick. Used the time to clean the house and write my
+book.
+```
 
-    jrnl yesterday: Called in sick. Used the time cleaning the house and writing my book.
-
-and hit return. `yesterday:` will be interpreted as a timestamp. Everything
-until the first sentence mark (`.?!`) will be interpreted as the title, the
-rest as the body. In your journal file, the result will look like this:
+`yesterday:` is  interpreted by `jrnl` as a timestamp. Everything until the
+first sentence ending (either `.`, `?`, or `!`) is interpreted as the title, and
+the rest as the body. In your journal file, the result will look like this:
 
     [2012-03-29 09:00] Called in sick.
-    Used the time cleaning the house and writing my book.
+    Used the time to clean the house and write my book.
 
-If you just call `jrnl`, you will be prompted to compose your entry - but you
-can also configure _jrnl_ to use your external editor.
+Entering `jrnl` without any arguments launches an external editor where you can
```

Comment:
I think it's good to keep the documentation more conversational here as well--this reads a bit like formal documentation but we want the readme to be as readable as possible.